next/app/layout.jsx

```diff
@@ -0,0 +1,59 @@
+import { Inter } from "next/font/google";
+import "./globals.css";
+import Script from "next/script";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <Script id="show-banner">
+          {`
+            (function () {
```

Comment:
There are two types which we can use for Nextjs routes (Both are supported)
 - Pages Router
 - App Router(The one you are using).
 
 @hs-saunved @dv-Gogna 
 
You can consider adding a separate example for the Nextjs Pages router
The place to add a script tag will differ.

Ref: https://nextjs.org/docs#app-router-vs-pages-router

Reply:
Yes
- But now almost every app uses this App router
- We can basically add this as info in the readme file 
- If anyone is still using that we he can refer from there

Reply:
Yes, Adding that in Readme also works.

You can also consider adding a gist of webchat integration in Readme for both.